internal/scm/bitbucketcloud/bitbucket_cloud.go

```diff
@@ -225,11 +252,11 @@ func (bbc *BitbucketCloud) GetOpenPullRequest(ctx context.Context, repo scm.Repo
 
 func (bbc *BitbucketCloud) MergePullRequest(ctx context.Context, pr scm.PullRequest) error {
 	bbcPR := pr.(pullRequest)
-	repoSlug := strings.Split(bbcPR.guiURL, "/")
+	repoSlug := strings.Split(bbcPR.guiURL, "/")[4]
```

Comment:
Can we use type assertions here?

Reply:
Ill give that a try

Reply:
@ngandhi64 what is the equivalent for the slug name?

Reply:
Would it be the repoName?

Reply:
I think I initially didn't know if repoName would work for the repoSlug, if we think that should work I can switch it

Reply:
Ill try using `bbcPR.prRepoName`

Reply:
Unfortunate @ngandhi64 I get back a 404 error trying to use the type assertion value, in this case it won't work and we'll need to use the split guiURL. I tried to use `bcPR.prRepoName` as the slug and it didn't work, I think because were using `bbcPR` in this case and not the repo we can't use the name as the slug. It worked for repos but not PRs, I think this is ok though because I am using a well named variable.